pkg/envtest/server.go

```diff
@@ -279,6 +302,39 @@ func (te *Environment) Start() (*rest.Config, error) {
 	return te.Config, err
 }
 
+//AddAPIServerUser helper function for adding APIServer user. Should be using while ControlPlane NOT running
+func (te *Environment) AddAPIServerUser(username string, groups []string) (*integration.User, error) {
+	if te.ControlPlane.IsStarted() {
+		return nil, fmt.Errorf("users cannot be added while ControlPlane running")
+	}
+
+	if te.KubeAPIServerUsers == nil {
+		te.KubeAPIServerUsers = make(map[string]*integration.User)
+	}
+	newUser := integration.User{
+		Token:  uuid.New().String(),
+		Name:   username,
+		UID:    username,
```

Comment:
i'm guessing this does not need to be a numeric value?

(i see it's a string in the struct, just curious)

Reply:
Should be a string, as I understand from doc:
https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authentication-strategies

>UID: a string which identifies the end user and attempts to be more consistent and unique than username.